theatre/studio/src/panels/SequenceEditorPanel/DopeSheet/Right/BasicKeyframedTrack/KeyframeEditor/BasicKeyframeConnector.tsx

```diff
@@ -84,17 +77,19 @@ const BasicKeyframeConnector: React.VFC<IBasicKeyframeConnectorProps> = (
   }
 
   return (
-    <ConnectorLine
-      ref={nodeRef}
-      connectorLengthInUnitSpace={connectorLengthInUnitSpace}
-      {...themeValues}
-      openPopover={(e) => {
-        if (node) openPopover(e, node)
-      }}
-    >
-      {popoverNode}
+    <>
+      <ConnectorLine
+        ref={nodeRef}
+        connectorLengthInUnitSpace={connectorLengthInUnitSpace}
+        {...themeValues}
+        openPopover={(e) => {
+          if (node) openPopover(e, node)
+        }}
+      >
+        {popoverNode}
+      </ConnectorLine>
       {contextMenu}
-    </ConnectorLine>
+    </>
```

Comment:
Interesting that the `{contextMenu}` is outside the area of `<ConnectorLine>` where `popoverNode` is. Does this need a comment describing why these two seemingly similar things are in different places?